oneapp-server/types/Application.js

```diff
@@ -45,6 +45,7 @@ const resolvers = {
 
       if (!isNil(input.foodStampInfo)) {
         logger.debug('Application (%s) update contains foodStampInfo', APPLICATION_NUMBER);
+        await dataSources.FoodStampInfoDao.updateFoodStampInfo(input.foodStampInfo);
```

Comment:
`applicationUpdate` will return an Application type from the mutation, so we should call `getFoodStampInfo` after the update and append to an object that will be returned.

Do you have any suggestions on how we can do this asynchronously? Each of the sections (`foodStampInfo`, `programInfo`, etc can be run simultaneously, so we could parallelize it so each of the sections run their update query and get query in parallel.